packages/matrix-identity-server/src/invitation/index.ts

```diff
@@ -134,6 +173,7 @@ const StoreInvit = <T extends string = never>(
   return (req, res) => {
     idServer.authenticate(req, res, (_data, _id) => {
       jsonContent(req, res, idServer.logger, (obj) => {
+        // eslint-disable-next-line @typescript-eslint/no-misused-promises
         validateParameters(res, schema, obj, idServer.logger, async (obj) => {
           const _address = (obj as storeInvitationArgs).address
```

Comment:
I think we shouldn't use the same fields name `address` for a phone number

Reply:
Should we create a different endpoint for phone maybe ?

Reply:
Can also be a different schema. If type is email, address is required, else phone is required

Reply:
But then the token I store will have two fields (phone and address) and one of them will be empty. Is it ok ?

Reply:
Are we talking about database schema or API field names ?

Reply:
I am now talking about how I store the invitation in the db, should I do the same thing as for the API field names, ie add a new column phone which will sometimes be empty, or keep one column for addresses which will store email as well as phone numbers ?

Reply:
I was talking about API fieldnames. For DB, do what is useful for you

Reply:
In the synapses db, the convention is to use a medium and an address, address being either an email address or a phone number. I will stick to this usage